infrastructure-scripts/claude_mcp.sh

```
@@ -332,8 +425,46 @@ if EXISTING_SERVERS=$(claude mcp list 2>&1); then
     log_with_timestamp "Found $EXISTING_COUNT existing MCP servers"
 
     if [ "$EXISTING_COUNT" -gt 0 ]; then
-        echo -e "${BLUE}📋 Currently installed servers:${NC}"
-        echo "$EXISTING_SERVERS" | head -10
+        echo -e "${BLUE}📋 Running parallel health checks on existing servers...${NC}"
+        
+        # Extract server names and start parallel health checks
+        SERVER_NAMES=($(echo "$EXISTING_SERVERS" | grep -E "^[a-zA-Z].*:" | cut -d':' -f1))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)